protected/humhub/config/web.php

```diff
@@ -15,23 +15,26 @@
     'components' => [
         'request' => [
             'class' => 'humhub\components\Request',
+            'csrfCookie' => ['httpOnly' => true, 'secure' => (isset(Yii::$app->request->isSecureConnection) ? false : true)],
```

Comment:
Don't like because.
You use `Yii::$app->request` in config for `Yii::$app->request`...
Probable better add this check to EVENT BEFORE_REQUEST and configure it in there `$event->sender->getRequest()...`